### PR TITLE
Avator/alerts

### PR DESCRIPTION
**Pull request by developer-anand**
# 1. Avatar, Navbar Components section component added 

##  Avatar, Navbar Components section added please give you feedback.

### CODE REVIEW

**Review by sanu29 — APPROVED** [review]
Great work. Some suggestions.
Naming should be consistent as if you use avt-img use avt everywhere.
File structure is good. Extra comment could be removed. 
Deployed link looks great just font size is very small.
Don't import each component CSS in documentation merge it in a file and import that CSS file.
Rest great work.

File: components/avatar/avatar.css [hunk 1]
```diff
@@ -0,0 +1,108 @@
...
+}
+
+.badge {
+  display: inline-flex;
+  position: relative;
+  width: 3rem;
+  margin: 1.6rem;
+}
+
+.badge-online {
```
Comment (humzahasan) [inline components/avatar/avatar.css]: Add a common class for badge

File: components/avatar/avatar.css [hunk 2]
```diff
@@ -0,0 +1,108 @@
+@import "/common.css";
+
+.avatar-container {
```
Comment (humzahasan) [inline components/avatar/avatar.css]: seperate css for documentation

Comment (developer-anand) [reply components/avatar/avatar.css]: Done can you check

File: components/avatar/avatar.css (hunk 1 above)
Comment (developer-anand) [reply components/avatar/avatar.css]: Added badge-status common class. Thanks for pointing.

**Comment by humzahasan** [comment]
Looks Good! Okay to merge

**Review by humzahasan — APPROVED** [review]
Good to GO!

**Comment by developer-anand** [comment]
Changed master to dev

**Pull request closed without merging**